src/product.njk

```diff
@@ -56,29 +56,38 @@ heroLg: true
                 <h5>Ticket-Based Support</h5>
             </div>
         </div>
-        <div class="mt-2"><p class="opacity-70">(some features coming soon)</p></div>
     </div>
 </div>
 
 {% include "components/divider-flow--top-red--light.njk" %}
 
-<!-- Feature List -->
-{% set cloudFeaturesHtml %}
-<div class="max-w-full overflow-x-scroll">
-    {% include "feature_lists/cloud.njk" %}
-</div>
-{% endset -%}
-
-{% set selfFeaturesHtml %}
-<div class="max-w-full overflow-x-scroll">
-    Hello Self Managed
-</div>
-{% endset -%}
+<div class="w-full ff-bg-light pb-32">
+    <div class="m-auto max-w-4xl">
+    	<div class="text-left">
+    		<h1>Sub-Minute setup</h1>
+	    	<p>With FlowForge deploying a new Node-RED project takes less than a minute. No configuration of proxies and setting up DNS, get going in less time it takes to make a cup of tee.</p>
+    	</div>
+    	<div class="text-right">
+		<h1>Entreprise-Ready</h1>
+		<p>Stacks, templates, etc
 
-
-<div class="about w-full ff-bg-light pb-32">
-    <div class="container m-auto text-center max-w-4xl">
-        <!-- Cloud Feature List -->
-        <div id="tab-cloud" class="ff-tab-content ff--active">{{ cloudFeaturesHtml | safe }}</div>
+		With centralized user management, and role based access control RBAC and centrilized users</p>
+	</div>
+    	<div class="text-left">
+		<h1>Team-Based Collaboration</h1>
+		<p></p>
+	</div>
+	<div class="text-right">
+		<h1>Node Catalog</h1>
+		<p>Leverage over 3000 custom nodes already developed to connect the physical world to digital servives and back again.</p>
+	</div>
+    	<div class="text-left">
+		<h1>Team-Based Collaboration</h1>
+		<p>Centralized user management to collaborate with multiple users</p>
+	</div>
+	<div class="text-right">
+		<h1>Audit Log</h1>
```

Comment:
@sammachin There's no documentation on this feature, can you add this?

Reply:
yes I'll add an inital section now, will probably need to evolve it as we gain more events in the audit log

Reply:
Joe was starting a list of Audit Events (https://docs.google.com/spreadsheets/d/1e1tHRGI01J65iTetjWcJj2sMVXYVm7kaIg4y_VnGQno/edit#gid=0)